Update search_syntax.md (#26538)
DataDog/documentation: content/en/continuous_integration/explorer/search_syntax.md

* `web*` matches all log messages starting with `web`.
* `*web` matches all log messages that end with `web`.

> **Note**: Wildcards only work as wildcards outside of double quotes. For example, `@ci.pipeline.name:"*test*"` matches a pipeline which contains the string `*test*` in its name. `@ci.pipeline.name:*test*` matches a pipeline which contains the string test in its name.

Reviewer: ```suggestion
**Note**: Wildcards only work as wildcards outside of double quotes. For example, `@ci.pipeline.name:"*test*"` matches a pipeline whose name contains the string `*test*`, whereas `@ci.pipeline.name:*test*` matches a pipeline whose name contains the string `test`.
```
